src/BPEssentials/Commands/Pay/Pay.cs

```diff
@@ -0,0 +1,28 @@
+﻿using BPEssentials.Abstractions;
+using BPEssentials.ExtensionMethods;
+using BPEssentials.Utils;
+
+using BrokeProtocol.Entities;
+using BrokeProtocol.Utility;
+using System.Linq;
+
+
+namespace BPEssentials.Commands
+{
+    public class Pay : Command
+    {
+        public void Invoke(ShPlayer player, ShPlayer target, int amount=1)
+        {
+            var item = Core.Instance.EntityHandler.Items.FirstOrDefault(x => x.Key == "Money");
+            if (player.MyItemCount(item.Value.index) < amount)
+            {
+                player.TS("you_dont_own", amount.ToString());
+                return;
+            }
+            player.TransferItem(DeltaInv.RemoveFromMe, item.Value.index, amount, true);
+            target.TransferItem(DeltaInv.AddToMe, item.Value.index, amount, true);
```

Comment:
The same also applies on these methods.
```suggestion
            player.TransferMoney(DeltaInv.RemoveFromMe, amount, true);
            target.TransferMoney(DeltaInv.AddToMe, amount, true);
```
Source code for said methods are:
```csharp
		// Token: 0x06001A55 RID: 6741 RVA: 0x00064554 File Offset: 0x00062754
		public void TransferMoney(byte deltaType, int amount = 1, bool dispatch = true)
		{
			this.TransferItem(deltaType, this.manager.money.index, amount, dispatch);
		}
```